doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,153 @@
+.. _governance:
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Author: Josh Warner
+:Author: François Boulogne
+:Author: Emmanuelle Gouillart
+:Author: Mark Harfouche
+:Author: Lars Grüter
+:Author: Egor Panfilov
+:Status: Draft
+:Type: Process
+:Created: 2019-07-02
+:Resolved:
+:Resolution:
+:Version effective: 0.16
+
+===========================================
+scikit-image governance and decision-making
+===========================================
+
+The purpose of this document is to formalize the governance process used by the
+scikit-image project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
```

Comment:
This is a marker/reminder to resolve "decision making process" for normal community members.

Reply:
improved below, under "Contributors"